site-manage/pageorder/tool/src/java/org/sakaiproject/site/tool/helper/order/impl/SitePageEditHandler.java

```diff
@@ -151,6 +156,11 @@ public void init() {
                 unhideables.add(toolIds[i].trim());
             }
         }
+        String uneditablesConfig = serverConfigurationService.getString(UNEDITABLES_CFG, "");
```

Comment:
Doesn't ServerConfigurationService have a getStrings method to handle this for you?